misc/travis-check.sh

```diff
@@ -61,6 +61,11 @@ if [ "$TARGET" = "Unix" ]; then
         )
 
     else
+		# For forcing make command to run update-docs target,
+		# call touch command and sleep command here.
+		touch man/*.rst
+		sleep 1
+		touch man/*.rst.i
```

Comment:
Isn't this a mistake of `touch man/*.rst.in`?

Reply:
Or, how about using `make -B`?

Reply:
Thank you. -B looks better. I will update this pull request after some testing with -B option.

Reply:
Updated. It works fine.